homeassistant/components/here_weather/const.py

```diff
@@ -0,0 +1,511 @@
+    "city": {"name": "City", "unit_of_measurement": None, "device_class": None},
+    "latitude": {"name": "Latitude", "unit_of_measurement": None, "device_class": None},
+    "longitude": {
+        "name": "Longitude",
+        "unit_of_measurement": None,
+        "device_class": None,
+    },
+    "utcTime": {
+        "name": "UTC Time",
+        "unit_of_measurement": None,
+        "device_class": DEVICE_CLASS_TIMESTAMP,
+    },
+}
+
+COMMON_ATTRIBUTES: dict[str, dict[str, str | None]] = {
+    "daylight": {"name": "Daylight", "unit_of_measurement": None, "device_class": None},
+    "description": {
+        "name": "Description",
+        "unit_of_measurement": None,
+        "device_class": None,
+    },
+    "skyInfo": {"name": "Sky Info", "unit_of_measurement": None, "device_class": None},
+    "skyDescription": {
+        "name": "Sky Description",
+        "unit_of_measurement": None,
+        "device_class": None,
+    },
+    "temperatureDesc": {
+        "name": "Temperature Description",
+        "unit_of_measurement": None,
+        "device_class": None,
+    },
+    "comfort": {
+        "name": "Comfort",
+        "unit_of_measurement": TEMP_CELSIUS,
+        "device_class": DEVICE_CLASS_TEMPERATURE,
+    },
+    "humidity": {
+        "name": "Humidity",
+        "unit_of_measurement": PERCENTAGE,
+        "device_class": DEVICE_CLASS_HUMIDITY,
+    },
+    "dewPoint": {
+        "name": "Dew Point",
+        "unit_of_measurement": TEMP_CELSIUS,
+        "device_class": DEVICE_CLASS_TEMPERATURE,
+    },
+    "precipitationProbability": {
+        "name": "Precipitation Probability",
+        "unit_of_measurement": PERCENTAGE,
+        "device_class": None,
+    },
+    "precipitationDesc": {
+        "name": "Precipitation Description",
+        "unit_of_measurement": None,
+        "device_class": None,
+    },
+    "airDescription": {
+        "name": "Air Description",
+        "unit_of_measurement": None,
+        "device_class": None,
+    },
+    "windSpeed": {
+        "name": "Wind Speed",
+        "unit_of_measurement": SPEED_KILOMETERS_PER_HOUR,
+        "device_class": None,
+    },
+    "windDirection": {
+        "name": "Wind Direction",
+        "unit_of_measurement": DEGREE,
+        "device_class": None,
+    },
+    "windDesc": {
+        "name": "Wind Description",
+        "unit_of_measurement": LENGTH_CENTIMETERS,
+        "device_class": None,
+    },
+    "windDescShort": {
+        "name": "Wind Description Short",
+        "unit_of_measurement": LENGTH_CENTIMETERS,
+        "device_class": None,
+    },
+    "icon": {"name": "Icon", "unit_of_measurement": None, "device_class": None},
+    "iconName": {
+        "name": "Icon Name",
+        "unit_of_measurement": None,
+        "device_class": None,
+    },
+    "iconLink": {
+        "name": "Icon Link",
+        "unit_of_measurement": None,
+        "device_class": None,
+    },
+}
+
+NON_OBERSATION_ATTRIBUTES: dict[str, dict[str, str | None]] = {
```

Comment:
```suggestion
NON_OBSERVATION_ATTRIBUTES: dict[str, dict[str, str | None]] = {
```

Reply:
Done in https://github.com/home-assistant/core/pull/28910/commits/8f53aa92dd0f4d457d2f41ffbd18849278ddbd1b